tdw_physics/util.py

```diff
@@ -9,6 +10,15 @@
 for filename in ModelLibrarian.get_library_filenames():
     MODEL_LIBRARIES.update({filename: ModelLibrarian(filename)})
 
+def xyz_to_arr(xyz : dict):

```

Comment:
Remove these functions. See `TDWUtils.array_to_vector3()` and `TDWUtils.vector3_to_array()`